diff --git a/node_modules/@emotion/styled/node_modules/@emotion/serialize/dist/emotion-serialize.cjs.dev.js b/node_modules/@emotion/styled/node_modules/@emotion/serialize/dist/emotion-serialize.cjs.dev.js
index ec6619f..aa3604c 100644
--- a/node_modules/@emotion/styled/node_modules/@emotion/serialize/dist/emotion-serialize.cjs.dev.js
+++ b/node_modules/@emotion/styled/node_modules/@emotion/serialize/dist/emotion-serialize.cjs.dev.js
@@ -88,6 +88,9 @@ function handleInterpolation(mergedProps, registered, interpolation) {
   }
 
   if (interpolation.__emotion_styles !== undefined) {
+    if(process.env.NODE_ENV === 'test') {
+      return null
+    }
     if (process.env.NODE_ENV !== 'production' && interpolation.toString() === 'NO_COMPONENT_SELECTOR') {
       throw new Error('Component selectors can only be used in conjunction with @emotion/babel-plugin.');
     }
